__tests__/app.comments.test.js

```diff
@@ -0,0 +1,66 @@
+const request = require("supertest");
+const app = require("../app");
+
+const testData = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET", () => {
+  test(`200: /api/articles/:article_id/comments - Responds with
+  an array of comments for the given article_id. Each comment has properties:
+    comment_id
+    votes
+    created_at
+    author which is the username from the users table
+    body`, () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({body}) => {
```

Comment:
You need a length check to make sure you're only getting back the amount of comments, for the article requested. 
At the moment, you could be getting back all the comments, and this test wouldnt fail